pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -120,6 +125,10 @@ type TektonHubStatus struct {
 	// The current installer set name
 	// +optional
 	HubInstallerSet map[string]string `json:"hubInstallerSets,omitempty"`
+
+	// The Base64 Encode data info of the Custom Logo
+	// +optional
+	CustomLogoInfo CustomLogo `json:"customLogoInfo,omitempty"`
```

Comment:
IMO The name `CustomLogo` (json: customLogo) looks better than `CusotmLogoInfo` 
may be you can change to,
```
type CustomLogoSpec struct {
	Base64Data string `json:"base64Data,omitempty"`
	MediaType  string `json:"mediaType,omitempty"`
}
```
and 
```
CustomLogo CustomLogoSpec `json:"customLogo,omitempty"`
```